src/include/ipc/info.h

```diff
@@ -107,7 +107,7 @@ struct sof_ipc_window_elem {
 struct sof_ipc_window {
 	struct sof_ipc_ext_data_hdr ext_hdr;
 	uint32_t num_windows;
-	struct sof_ipc_window_elem window[];
+	struct sof_ipc_window_elem window[7];
```

Comment:
Do you need to change this ?

Reply:
Used toolchain reject packing variable-size structures inside another one, even as last field - same situation as it was before for CC_VERSION